src/asm/asm.s

```diff
@@ -94,6 +94,12 @@ _x86_64_asm_invlpg:
     invlpg (%rdi)
     retq
 
+.global _x86_64_asm_invpcid
+.p2align 4
+_x86_64_asm_invpcid:
+    invpcid (%rsi), %rdi
```

Comment:
Are you sure about the argument order here? I'm mostly using intel syntax, so I'm not really familiar with AT&T syntax. I know that the source and destination arguments are swapped with AT&T, but here we have two input arguments, so I'm not sure.

Reply:
This is correct, [per the INVPCID docs](https://www.felixcloutier.com/x86/invpcid) the first command is a register (containing the kind), the second is a memory location (containing the command), so the instruction is `invpcid rdi, [rsi]`. 

AT&T syntax is backwards, so the above code is right.

Reply:
Thanks for checking!